pkg/query-service/rules/manager.go

```diff
@@ -223,7 +221,6 @@ func NewManager(o *ManagerOptions) (*Manager, error) {
 		prepareTaskFunc:     o.PrepareTaskFunc,
 		prepareTestRuleFunc: o.PrepareTestRuleFunc,
 		alertmanager:        o.Alertmanager,
-		sqlstore:            o.SQLStore,
```

Comment:
You cannot remove this here. If you remove this here, you must also remove it from the struct, at which point you realize that it's required as the other components are still relying on SQLStore.